xml/favicons.xml

```diff
@@ -2367,4 +2376,4 @@
     <param pos="0" name="hw.vendor" value="FatPipe Networks"/>
   </fingerprint>
 
-</fingerprints>
\ No newline at end of file
+</fingerprints>
```

Comment:
It appears a newline was introduced at the end of the file. Please remove for consistency across fingerprint files.